src/layouts/Layout.astro

```diff
@@ -29,6 +29,7 @@ const { title } = Astro.props
         <nord-layout>
             <NavGroup />
             <nord-header slot="header">
+                {/* TODO : to replace by a search bar on desktop */}
```

Comment:
I use only nord component, you can check this component : https://nordhealth.design/components/command-menu/

Reply:
I will do it after this PR if you wnat.

Reply:
Yes. You can use `pagefind` for example to create a search index from Astro files.